src/client/components/projectDetails.jsx

```diff
@@ -15,6 +15,29 @@ class ProjectDetails extends React.Component {
     this.startBuild = this.startBuild.bind(this);
   }
 
+  customBuild(build){
+    let path = window.prompt(`Enter the name of the branch/tag you need to build.
+
+      Format:
+
+        "Any repo: Any branch/ tag"
+
+      Examples:
+
+        raven: latest
```

Comment:
Roger handles it.. here 👇
 https://github.com/namshi/roger/blob/master/src/client/actions/actions.js#L11